lib/shopify-cli/commands/config.rb

```diff
@@ -71,6 +72,29 @@ def call(_args, _name)
           end
         end
       end
+
+      class ShopifolkBeta < ShopifyCli::SubCommand
+        options do |parser, flags|
+          parser.on('--enable') { flags[:action] = 'enable' }
+          parser.on('--disable') { flags[:action] = 'disable' }
+          parser.on('--status') { flags[:action] = 'status' }
+        end
+
+        def call(_args, _name)
+          is_enabled = ShopifyCli::Config.get_bool('shopifolk-beta', 'enabled')
+          if options.flags[:action] == 'disable' && is_enabled
+            ShopifyCli::Config.set('shopifolk-beta', 'enabled', false)
+            @ctx.puts(@ctx.message('core.config.shopifolk-beta.disabled'))
+          elsif options.flags[:action] == 'enable' && !is_enabled
+            ShopifyCli::Config.set('shopifolk-beta', 'enabled', true)
+            @ctx.puts(@ctx.message('core.config.shopifolk-beta.enabled'))
```

Comment:
Just noticed, theres no corresponding message for these outputs

Reply:
They seem to map onto `core.config.feature`, seems confusing though... hmm.

Reply:
Ah, I was running `shopify config feature shopifolk-beta` instead of `shopify config shopifolk-beta`.

Reply:
Fixed in f06c6cf0